src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/Amd64/GcInfo.cs

```diff
@@ -259,6 +259,14 @@ public override string ToString()
 
                 sb.AppendLine($"    Has Tailcalls: {_wantsReportOnlyLeaf}");
             }
+            else if (_machine == (Machine) 0x6264) /* LoongArch64 */
```

Comment:
@cshung Can we bump the System.Reflection.Metadata dependency to 7.0.0 to avoid the hardcoded constants?

Reply:
I'm afraid that some dependencies are pinned due to usage of this R2RDump code in ILSpy but let's hear what Andrew says.

Reply:
@eduardo-vp, can you please work with @christophwille to see if it is feasible to do the upgrade?

Reply:
Sure, I'll take a look

Reply:
> @eduardo-vp, can you please work with @christophwille to see if it is feasible to do the upgrade? Meanwhile, feel free to get unblocked with the constants. Flowing dependencies across repos can take a while.

We just gave it a try - we can upgrade ILSpy to SRM 7 and still keep .Decompiler at SRM 6.

Reply:
@cshung and I just doubled checked that it's ok to upgrade to SRM 7.0.0

Reply:
I created https://github.com/dotnet/runtime/pull/85738 to perform the upgrade.

Reply:
The upgrade is merged, feel free to convert the hard code constants to symbolic constants.

Reply:
OK, Thanks~